test/src/unit-capi-array.cc

```diff
@@ -2235,7 +2237,9 @@ TEST_CASE_METHOD(
 }
 
 TEST_CASE_METHOD(
-    ArrayFx, "Test array serialization", "[array][serialization]") {
+    ArrayFx,
+    "Test array serialization",
+    "[array][serialization][array-schema-evolution]") {
```

Comment:
Tag this with rest too?

Reply:
@jp-dark has written a `[rest]` test that reproduces this issue in: https://github.com/TileDB-Inc/TileDB/pull/5142
 
Those tests would need more adaptation than simply adding a tag to run on all VFSs etc, but I don't think there's any added value here. IMO we should just close this PR.

Reply:
@jp-dark Let me know if you don't agree with @ypatia to close this PR.

Reply:
I'll go ahead and close it.